.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ parameters:
 executors:
     main-env:
         docker:
-            - image: ucbbar/chipyard-image:1.0.1
+            - image: ucbbar/chipyard-ci-image:84cfc73
```

Comment:
For this PR, can you revert this change. Just modify the `timeout` settings and not this image.

Reply:
Reverted docker change, and re-changed lines 115 and 234 to use 20m timeout